std/algorithm/searching.d

```diff
@@ -1204,6 +1210,75 @@ bool endsWith(alias pred, R)(R doesThisEnd)
     }
 }
 
+/**
```

Comment:
OK, could leave as-is. (I wasn't suggesting removing the comment, just the doccommentness).